src/python_testing/TC_ACE_1_5.py

```diff
@@ -0,0 +1,164 @@
+from mobly import asserts
+
+
+class TC_ACE_1_5(MatterBaseTest):
+
+    def OpenCommissioningWindow(self) -> CommissioningParameters:
+        try:
+            params = self.th1.OpenCommissioningWindow(
+                nodeid=self.dut_node_id, timeout=600, iteration=10000, discriminator=self.matter_test_config.discriminators[0], option=1)
+            time.sleep(5)
+            return params
+
+        except Exception as e:
+            logging.exception('Error running OpenCommissioningWindow %s', e)
+            asserts.assert_true(False, 'Failed to open commissioning window')
+
+    async def read_currentfabricindex_expected_success(self, th) -> int:
+        cluster = Clusters.Objects.OperationalCredentials
+        attribute = Clusters.OperationalCredentials.Attributes.CurrentFabricIndex
+        current_fabric_index = await self.read_single_attribute_check_success(dev_ctrl=th, endpoint=0, cluster=cluster, attribute=attribute)
+        return current_fabric_index
+
+    async def write_acl(self, acl, th):
+        # This returns an attribute status
+        result = await th.WriteAttribute(self.dut_node_id, [(0, Clusters.AccessControl.Attributes.Acl(acl))])
+        asserts.assert_equal(result[0].Status, Status.Success, "ACL write failed")
+
+    async def read_descriptor_expect_success(self, th):
+        cluster = Clusters.Objects.Descriptor
+        attribute = Clusters.Descriptor.Attributes.DeviceTypeList
+        await self.read_single_attribute_check_success(
+            dev_ctrl=th, endpoint=0, cluster=cluster, attribute=attribute)
+
+    async def read_basic_expect_success(self, th):
+        cluster = Clusters.Objects.BasicInformation
+        attribute = Clusters.BasicInformation.Attributes.VendorID
+        await self.read_single_attribute_check_success(
+            dev_ctrl=th, endpoint=0, cluster=cluster, attribute=attribute)
+
+    async def read_basic_expect_unsupported_access(self, th):
+        cluster = Clusters.Objects.BasicInformation
+        attribute = Clusters.BasicInformation.Attributes.VendorID
+        await self.read_single_attribute_expect_error(
+            dev_ctrl=th, endpoint=0, cluster=cluster, attribute=attribute, error=Status.UnsupportedAccess)
+
+    async def read_descriptor_expect_unsupported_access(self, th):
+        cluster = Clusters.Objects.Descriptor
+        attribute = Clusters.Descriptor.Attributes.DeviceTypeList
+        await self.read_single_attribute_expect_error(
+            dev_ctrl=th, endpoint=0, cluster=cluster, attribute=attribute, error=Status.UnsupportedAccess)
+
+    @async_test_body
+    async def test_TC_ACE_1_5(self):
+        self.print_step(1, "Comissioning, already done ")
+        self.th1 = self.default_controller
+
+        self.print_step(2, "TH1 opens the commissioning window on the DUT")
+        params = self.OpenCommissioningWindow()
+
+        new_certificate_authority = self.certificate_authority_manager.NewCertificateAuthority()
```

Comment:
As with the open commissioning window, we're probably going to get enough tests that require this that it's worthwhile abstracting into the base.

ie, some function that creates a controller on a new fabric and commissions all in one go.

Reply:
For now I just left a `TODO` msg but in the next one changes I will make sure that this is included.